react/features/base/responsive-ui/components/DimensionsDetector.native.js

```diff
@@ -46,9 +53,11 @@ export default class DimensionsDetector extends PureComponent<Props> {
      * @returns {void}
      */
     _onLayout({ nativeEvent: { layout: { height, width } } }) {
-        const { onDimensionsChanged } = this.props;
+        const { onDimensionsChanged, _clientResized } = this.props;
 
         onDimensionsChanged && onDimensionsChanged(width, height);
+        _clientResized(width, height);
```

Comment:
Why is this needed? The action is already dispatched here: https://github.com/jitsi/jitsi-meet/blob/8da154b185c81b3f9f856b88d1fbe7ba25acda6f/react/features/app/components/App.native.js#L187 and reduced here: https://github.com/jitsi/jitsi-meet/blob/8da154b185c81b3f9f856b88d1fbe7ba25acda6f/react/features/base/responsive-ui/reducer.js#L25

Moreover, this class was designed to be generic so you could use to measure any element, not necessarily just the entire app, hence why it's decoupled from the store. I understand that might not be clear at first sight, sorry about that!

Reply:
My bad. Did not followed the `onDimensionsChanged` callback.